### DISCUSSION TITLE
%homedrive% and %homepath% vs %userprofile%

### DISCUSSION THREAD

**Discussion by tonypags** [PowerShellOrg/Plaster#401] [General]
The parameters for `FullName` and `Email` are read from the `.gitconfig` file of the user. The issue I have seen is that Plaster looks under `%userprofile%` but sometimes these files are stored elsewhere because git looks at `%homedrive%` and `%homepath%`. 

Because it's possible to store the previous values entered, this is a very minor issue. However, you may want to change how plaster looks for users' git files.

**Comment by rkeithhill** [comment]
I suppose we could switch to use `%homedrive%%homepath%` on Windows to find the file if it's not found under `%userprofiile%`. Do you think that would be sufficient?  Another is to run `git config --global --get-regexp user.*`.
